### DISCUSSION TITLE
How to change mapping option label/value to id/name

### DISCUSSION THREAD

**Discussion by abinhho** [csandman/chakra-react-select#100] [Q&A] [answered]
Thank you for your nice contribution.

Default option type is

```js
{
   value: "coffee",
   label: "Coffee",
}
```

But I want to use like this

```js
{
   id: "coffee",
   name: "Coffee",
}
```

How can I do that?

**Comment by joao-salomao** [comment, marked as answer]
@abinhho you can use the props `getOptionLabel` and `getOptionValue` from [react select](https://react-select.com/props)

**Comment by csandman** [comment]
@joao-salomao Thanks for helping out here, and you're spot on.

Here's a full example using `getOptionLabel` and `getOptionValue`

```js
import { Select } from "chakra-react-select";

const options = [
  { id: "vanilla", name: "Vanilla" },
  { id: "chocolate", name: "Chocolate" },
  { id: "strawberry", name: "Strawberry" },
  { id: "salted-caramel", name: "Salted Caramel" }
];

const Example = () => (
  <Select
    isMulti
    name="flavors"
    options={options}
    placeholder="Select some flavors..."
    getOptionLabel={({ name }) => name}
    getOptionValue={({ id }) => id}
  />
);

export default Example;
```

CodeSandbox Link: https://codesandbox.io/s/chakra-react-select-custom-label-and-value-qn8je?file=/example.js
